bin/aurbuild

```diff
@@ -133,6 +133,10 @@ if [[ $build == build_chroot ]]; then
     sudo cp -v "$tmp"/pacman.conf "$CHROOT"/root/etc/pacman.conf
 fi
 
+# Canonicalise paths
+pool="$(realpath "${pool}")"
+root="$(realpath "${root}")"
```

Comment:
Use `realpath -e` to get built-in error reporting (all components of the path must exist), also the braces for pool/root are redundant here.